app/display_book.py

```diff
@@ -0,0 +1,16 @@
+from abc import ABC
+from app.book import Book
+
+
+class DisplayBook(ABC):
+    pass
+
+
+class DisplayBookInConsole(DisplayBook):
+    def __init__(self, book: Book) -> None:
```

Comment:
Why these actions do in init???
Init is used for construct objects, not for do business logic